.github/workflows/release-against-rancher.yml

```diff
@@ -3,15 +3,15 @@ on:
   workflow_dispatch:
     inputs:
       rancher_ref:
-        description: "Submit PR against the following rancher/rancher branch (e.g. release/v2.7)"
+        description: "Submit PR against the following rancher/rancher branch (e.g. release/v2.12)"
         required: true
-        default: "release/v2.10"
+        default: "release/v2.11"
```

Comment:
Since the description mentions v2.12 as an example, is v2.11 here deliberate? Or is it a typo?